### ISSUE TITLE
set `startDate` & `endDate` for daterangepicker

### ISSUE THREAD

**Issue by CitySim** [mpneuried/iggy#55]
currently the daterangepicker does not selects the correct "predefined range" (see http://www.daterangepicker.com/#ex4) this can be fixed by setting the `startDate` and `endDate` options. The fix could look something like this:

https://github.com/mpneuried/iggy/blob/master/_src/js/views/facets/daterange.coffee#L6

``` coffee
forcedDateRangeOpts: =>
  opens: "right"
  startDate: moment(@model.get("value")[0])
  endDate: moment(@model.get("value")[1])
```

**Comment by trietz** [comment]
still doesn't select the correct Timespan cause the startdate, enddate are not parsed with moment.

> forcedDateRangeOpts: =>
>   opens: "right"
>   startDate: moment(@model.get("value")[0])
>   endDate: moment(@model.get("value")[1])

**Comment by trietz** [comment]
Geht leider noch nicht... bitte nochmal korrigieren.

Siehe vorheriger comment.

Bei Bedarf bitte bei Sven melden.

**Comment by mpneuried** [comment]
fixed with 2143b450ed823b7c0c00dfaf551412a3230fbe74
i also added the option `dateformat` to define the given format.

e.g.:
```js
{
	type: "daterange",
	name: "issue55_frmt",
	label: "daterange55_frmt",
	dateformat: "DD [foo] MMM [bar] YY",
	value: [
		moment().add( -4, "d" ).format( "DD [foo] MMM [bar] YY" ),
		moment().add( 4, "d" ).format( "DD [foo] MMM [bar] YY" )
	]
}
```
Results in Facet view:
![bildschirmfoto 2017-01-25 um 13 24 59](https://cloud.githubusercontent.com/assets/300631/22290551/bc3a55f0-e301-11e6-84f8-e10438a7d9ad.png)

Range Picker View:
![bildschirmfoto 2017-01-25 um 13 27 50](https://cloud.githubusercontent.com/assets/300631/22290606/200484b6-e302-11e6-83e3-81ff42b73bcb.png)
